.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Eslint should highlight the _it, describe, assert_ functons as not defined in your test.js file.
To prevent this, you need to setup your eslint to understand mocha specific global variables.
It can be done by adding the mocha env into the .eslintrc file.

https://eslint.org/docs/user-guide/configuring#specifying-environments